iceoryx_utils/platform/win/include/iceoryx_utils/platform/win32_errorHandling.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+
+#define PrintLastErrorToConsole() __PrintLastErrorToConsole("", __FILE__, __LINE__)
+int __PrintLastErrorToConsole(const char* functionName, const char* file, const int line) noexcept;
+
+#define Win32Call(function)                                                                                            \
```

Comment:
is Win64 supported too?

Reply:
Ah i see its then consistent with _WIN32 macro, then it's ok

Reply:
oki doki